spec/expressions.md

```diff
@@ -3505,7 +3505,7 @@ bool? operator |(bool? x, bool? y);
 
 The following table lists the results produced by these operators for all combinations of the values `true`, `false`, and `null`.
 
-| `x`     | `y`     | `x & y` | <code>x &#124; y</code> |
+| `x`     | `y`     | `x & y` | `x | y` |
```

Comment:
This doesn't appear to render correctly.

Reply:
Yeah, the Docs preview in VS Code shows it correctly, but GitHub isn't, so I rebased it out